docs/content/v0.20/release.md

```diff
@@ -0,0 +1,84 @@
+
+The end-to-end tests include ones written in `bash`, and these are the only documentation telling a user how to use the present version of this repository. Again, these tests do not use any published artifacts from a release of this repo.
+
+We will introduce another category of tests, _release tests_. These test a given release, using the published artifacts of that release. These tests will be executables that a contributor invokes, passing the identifier of the desired release on the command line. We will create a GitHub workflow that can be explicitly triggered and will apply the release tests (in the branch that the workflow is directed at) to the latest release in that branch. We will also create a GitHub workflow that is simply told which release to test.
+
+We will maintain a document that lists releases that pass our quality bar. The latest of those is thus the latest stable release. This document is updated in `main` as quality evaluations come in.
+
+We will maintain an examples document that shows users how to exercise a few scenarios using the latest stable release.
+
+## Policy
+
+We aim for all regular releases to be working. In order to do that, we have to make test releases and test them. The widely recognized pattern for doing that is to make "release candidates" (i.e., releases for testing purposes) `1.2.3-rc0`, `1.2.3-rc1`, `1.2.3-rc2`, and so on, while trying to get to a quality release `1.2.3`. Once one of them is judged to be of passing quality, we make a release without the `-rc<N>` suffix. Due to the self-reference in the KCM PostCreateHook, this will involve making a new commit.
+
+When reaching a new major or minor version, we make a release branch for it. That is, a branch named `release-$major.$minor`. At first this branch differs from `main` in one commit, the one that sets the self-reference in the KCM PostCreateHook. After that, the branches continue to evolve in the usual ways.
+
+### Deliberately feature-incomplete releases
+
+We plan a few deliberately feature-incomplete releases. They will be regular releases as far as the technology here is concerned. They will be announced only to selected users who acknowledge that they are getting something that is incomplete. In GitHub, these will be marked as "pre-releases". The status of these releases will be made clear in their documentation (which currently appears in [the release notes](release-notes.md).
+
+## Step-by-Step
+
+Making a new release requires a contributor to do the following things. Here `$version` is the semver identifier for the release (e.g., `1.2.3-rc2`).
+
+- Edit [the source for the KCM PCH](../../../config/postcreate-hooks/kubestellar.yaml) and update the tag in the reference to the KCM container image (it appears in the last object, a `Job`).
```

Comment:
1. What is "KCM" ? 
2. Maybe more specific pointer, e.g., "Update the --version  argument"

Reply:
1. Is answered twice in this document already